CorsixTH/Lua/world.lua

```diff
@@ -2469,7 +2434,6 @@ function World:afterLoad(old, new)
     self:nextEmergency()
   end
   if old < 31 then
-    self.hours_per_day = 50
```

Comment:
Best to leave old modifications in, in case code in between relied on it. In this case it probably doesn't. Note that old saves will continue to have `self.hours_per_day` until you nil it out in your < 124 afterLoad.

Reply:
I fixed most of the issues. I kinda refactor the whole annoucments section in hospital, changing some of it's logic. It would be nice if you could have a look especially on that.